client/src/app/site/assignments/modules/assignment-poll/components/assignment-poll-vote/assignment-poll-vote.component.ts

```diff
@@ -308,11 +323,22 @@ export class AssignmentPollVoteComponent extends BasePollVoteComponent<ViewAssig
     }
 
     public saveGlobalVote(globalVote: GlobalVote, user: ViewUser = this.user): void {
-        this.voteRequestData[user.id].value = {};
```

Comment:
I changed the logic here because to me it seems like the wanted behavior is not achieved currently: With this line, the subsequent if-statement, which should introduce toggle-behavior, is always evaluated to false.